Doc/library/traceback.rst

```diff
@@ -212,10 +212,12 @@ The module also defines the following classes:
 :class:`TracebackException` objects are created from actual exceptions to
 capture data for later printing in a lightweight fashion.
 
-.. class:: TracebackException(exc_type, exc_value, exc_traceback, *, limit=None, lookup_lines=True, capture_locals=False)
+.. class:: TracebackException(exc_type, exc_value, exc_traceback, *, limit=None, lookup_lines=True, capture_locals=False, compact=False)
 
    Capture an exception for later rendering. *limit*, *lookup_lines* and
-   *capture_locals* are as for the :class:`StackSummary` class.
+   *capture_locals* are as for the :class:`StackSummary` class. If *compact*
+   is true, only data that is required for :method:`format` and
+   :method:`format_exception_only` is saved in the class attributes.
```

Comment:
Yes, though I'll avoid "suppresses" so as not to confuse with `__suppress_context__` which is applied in format().

I also don't need to mention format_exception_only because it's called from format.